docs/reference/commandline/node_ls.md

```diff
@@ -161,6 +166,11 @@ e216jshn25ckzbvmwlnh5jr3g: swarm-manager1 Ready
 35o6tiywb700jesrt3dmllaza: swarm-worker1 Needs Rotation
 ```
 
+To print all the nodes with json output, use `json` directive:
+```bash
```

Comment:
can you change this to `console`? (I'll be updating other occurrences, so I could do that at the same time if needed; some background why `console` here: https://github.com/docker/docker.github.io/pull/12425)

Reply:
Also for some of the other docs that were added

Reply:
Sure 👍